### PR TITLE
https://unit.atlassian.net/browse/NP-46407

### PR DESCRIPTION
**Pull request by StigNorland**

### CODE REVIEW

File: search-commons/src/main/java/no/unit/nva/search2/ticket/TicketQuery.java [hunk 1]
```diff
@@ -158,26 +188,31 @@ public TicketQuery withUser(String username) {
...
+     * @param userName current user
+     * @return TicketQuery (builder pattern)
      */
-    public TicketQuery withRequiredOrganization(URI organization) {
-        final var filter = new TermQueryBuilder(ORGANIZATION_ID_KEYWORD, organization.toString())
-            .queryName(ORGANIZATION + ID);
-        this.addFilter(filter);
+    public TicketQuery withFilterCurrentUser(String userName) {
+        this.currentUser = userName;
+        if (isUserOnly(ticketTypes)) {
```
Comment (StigNorland) [inline search-commons/src/main/java/no/unit/nva/search2/ticket/TicketQuery.java:211]: isNotCurrator ?

Comment (brinxmat) [reply search-commons/src/main/java/no/unit/nva/search2/ticket/TicketQuery.java:211]: isGeneralSupportCase?

**Review by ketilaa — APPROVED** [review]

File: search-commons/src/main/java/no/unit/nva/search2/ticket/TicketQuery.java [hunk 2]
```diff
@@ -158,26 +188,31 @@ public TicketQuery withUser(String username) {
...
+     * <p>This is to avoid the Query to return documents that are not available for the user.</p>
+     *
+     * @param userName current user
+     * @return TicketQuery (builder pattern)
      */
-    public TicketQuery withRequiredOrganization(URI organization) {
-        final var filter = new TermQueryBuilder(ORGANIZATION_ID_KEYWORD, organization.toString())
-            .queryName(ORGANIZATION + ID);
-        this.addFilter(filter);
+    public TicketQuery withFilterCurrentUser(String userName) {
```
Comment (brinxmat) [inline search-commons/src/main/java/no/unit/nva/search2/ticket/TicketQuery.java:209]: withFilterOwner?

**Comment by ketilaa** [comment]
A bit hard to see what you have introduced with this code. Some check for valid aggregation spec? I would have preferred a short description with the PR as to what you have done. The Jira issue number in the branch name have me a hint, but...

File: search-commons/src/main/java/no/unit/nva/search2/common/Query.java [hunk 3]
```diff
@@ -308,7 +308,7 @@ private Stream<Entry<K, QueryBuilder>> getQueryBuilders(K key) {
     private QueryBuilder multiMatchQuery(K searchAllKey, K fieldsKey) {
         var fields = fieldsToKeyNames(getValue(fieldsKey).toString());
         var value = getValue(searchAllKey).toString();
-        logger.info(value);
+        logger.debug(value);
```
Comment (anetteOlli) [inline search-commons/src/main/java/no/unit/nva/search2/common/Query.java:311]: debug? det forsvinner vel bare under kjøring på aws lambda?

**Review by anetteOlli — APPROVED** [review]

File: search-commons/src/main/java/no/unit/nva/search2/ticket/TicketQuery.java (hunk 1 above)
Comment (StigNorland) [reply search-commons/src/main/java/no/unit/nva/search2/ticket/TicketQuery.java:211]: I am guessing what kind of user based on accessrights...

**Pull request merged**